Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -50,6 +50,23 @@ Ref<Scrollbar> Scrollbar::createNativeScrollbar(ScrollableArea& scrollableArea,
     return adoptRef(*new Scrollbar(scrollableArea, orientation, size));
 }
 
+int Scrollbar::pixelsPerLineStep(int viewWidthOrHeight)
+{
+#if PLATFORM(GTK)
+    if (const char* scrollStep = getenv("WEBKIT_SCROLL_STEP")) {
+        int value;
+        if (sscanf(scrollStep, "%d", &value) == 1 && value > 0)
+            return value;
+    }
```

Comment:
This is great for testing, but let's take this out before landing to ensure applications do not try to use it.

Reply:
Yes, that's the plan.